MIGRATION.md

```diff
@@ -225,6 +228,64 @@ In the meantime, these migration notes are the best available documentation on t
 
 ### Breaking changes
 
+
+#### No more default export from `@storybook/addons`
+
+The default export from `@storybook/addons` has been removed. Please use the named exports instead:
+
+```js
+import { addons } from '@storybook/addons';
+```
+
```

Comment:
```suggestion

The named export has been available since 6.0 or earlier, so your updated code will be backwards-compatible with older versions of Storybook.
```